stdci.yaml

```diff
@@ -1,3 +1,6 @@
+# SPDX-FileCopyrightText: Red Hat, Inc.
+# SPDX-License-Identifier: GPL-2.0-or-later
```

Comment:
This file should be removed, it is part of the automation directory. But we can do this later.

Reply:
https://github.com/oVirt/vdsm/issues/327